src/pages/Election.tsx

```diff
@@ -131,7 +128,14 @@ const Information: FC<ElectionSubpage> = ({ id, election }) => {
       <br />
       <h4>Positions</h4>
       <br />
-      <PositionList election={election} />
+      <PositionList
+        election={election}
+        updateElection={() => {
+          if (updateElection) {
+            updateElection(election.id);
+          }
```

Comment:
You could probably remove the braces `{  }` on this and have it all on one line.